### ISSUE TITLE
bath vent cover fails to slice

### ISSUE THREAD

**Issue by rbrome** [Ultimaker/Cura#16705]
### Cura Version

5.4.0

### Operating System

macOS Ventura 13.5.2

### Printer

Ultimaker S5

### Name abnormal settings

no

### Describe model location

center, fails at any location or rotation

### Describe your model

first time tying

### Add your .zip here ⬇️

[UMS5_bath vent cover.3mf.zip](https://github.com/Ultimaker/Cura/files/12566700/UMS5_bath.vent.cover.3mf.zip)

**Comment by rbrome** [comment]
Huh. I created this in SketchUp, which did acknowledge the model as "solid" when I made it a component. I thought that meant it was "watertight". 
Regardless, I eventually discovered that the "non-watertight" model did print just fine as-is when I changed to the "Sprint" profile, which does increase the "Initial Layer Height" to 0.4. 
I did try rotating and moving the model as suggested. That did not seem to help in this case.
